recipe/build.sh

```diff
@@ -40,22 +40,40 @@ LINKFLAGS="${LINKFLAGS} -L${LIBRARY_PATH}"
 # https://stackoverflow.com/a/5244844/1005215
 sed -i.bak "s,cc,${TOOLSET},g" ${SRC_DIR}/project-config.jam
 
-./b2 -q \
-    variant=release \
-    address-model="${ARCH}" \
-    architecture=x86 \
-    debug-symbols=off \
-    threading=multi \
-    runtime-link=shared \
-    link=static,shared \
-    toolset=${TOOLSET}-custom \
-    include="${INCLUDE_PATH}" \
-    cxxflags="${CXXFLAGS}" \
-    cxxstd=14 \
-    linkflags="${LINKFLAGS}" \
-    --layout=system \
-    -j"${CPU_COUNT}" \
-    install | sed -e "s|${PREFIX}|<PREFIX>|g" | tee b2.log 2>&1
+if [[ "$cxx_compiler" == "toolchain_cxx" ]]; then
+    ./b2 -q \
+        variant=release \
+        address-model="${ARCH}" \
+        architecture=x86 \
+        debug-symbols=off \
+        threading=multi \
+        runtime-link=shared \
+        link=static,shared \
+        toolset=${TOOLSET}-custom \
+        include="${INCLUDE_PATH}" \
+        cxxflags="${CXXFLAGS}" \
+        cxxstd=14 \
```

Comment:
CXXFLAGS include `cxxstd=14`. Is this needed?

Reply:
Apparently so. We were having build issues without this.

Though if something else occurs to you, would be happy to hear. :)

xref: https://github.com/conda-forge/z5py-feedstock/pull/6

Reply:
Ok, I wasn't aware that this .
If cxx14 is in the cxxflags this should not be necessary.
But I am not sure if this is working properly, because I see the linker error with the new boost version here https://github.com/conda-forge/z5py-feedstock/pull/6.
The boost issue I found above suggests that not compiling with cxx 14 might be the issue,